backend/core/management/commands/watch_account_creation.py

```diff
@@ -0,0 +1,51 @@
+"""
+Watch and alert when a certain number of accounts are created with the same trusted domaine.
+"""
+from django.core.management.base import BaseCommand
+
+from core.lib.discord.api import write_in_channel
+from core.utils.email_domain import get_user_cnt_per_email_domain
+from core.utils.time import time_ago
+
+
+class Command(BaseCommand):
+    help = "Alert when a certain of accounts are created with the same trusted mail domain."
```

Comment:
In order to match the new top-level documentation string (if my previous suggestion is accepted).

```suggestion
    help = "Send an alert on Discord when X users have created accounts using" \
           " email addresses considered trusted during the last Y hours."
```